cmd/kurator/app/install/pixie/vizier/vizier.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vizier
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"kurator.dev/kurator/pkg/generic"
+	plugin "kurator.dev/kurator/pkg/plugin/pixie/vizier"
+)
+
+const (
+	communityCloudAddr = "withpixie.ai:443"
+	pxNamespace        = "px"
+)
+
+var pluginArgs = plugin.InstallArgs{}
+
+func NewCmd(opts *generic.Options) *cobra.Command {
+	vizierCmd := &cobra.Command{
+		Use:   "pixie-vizier",
+		Short: "Install pixie vizier component",
+		RunE: func(c *cobra.Command, args []string) error {
+			plugin, err := plugin.NewPlugin(opts, &pluginArgs)
+			if err != nil {
+				logrus.Errorf("pixie vizier init error: %v", err)
+				return fmt.Errorf("pixie init error: %v", err)
+			}
+
+			logrus.Debugf("start install pixie Global: %+v ", opts)
```

Comment:
it is to install pixier vizier.

And reduce print so much info, which is not that useful